src/Backend/SgateShopgatePlugin/Components/User.php

```diff
@@ -123,15 +123,6 @@ public function loginUser($request, $httpResponse)
             $httpResponse->setHttpResponseCode(401);
             $httpResponse->setBody(json_encode($error));
         } else {
-            if (!empty($basket['content'])) {
-                $this->basket->clearBasket();
-                $this->basket->sRefreshBasket();
-
-                foreach ($basket['content'] as $basketItem) {
-                    $this->basket->sAddArticle($basketItem['ordernumber'], $basketItem['quantity']);
-                }
-            }
-
```

Comment:
This change was in response to a bug which appeared during checkout: The articles were added to the cart but the custom attributes were lost. This was found to be caused by the marked code clearing the cart and adding all articles again, the second time without the attribute customization. Once the cart is cleared, the customized attributes can no longer be restored. 

As there is no obvious reason for this code, it was decided to remove it to fix the aforementioned bug.